ui/oopsyraidsy/oopsyraidsy.js

```diff
@@ -304,6 +309,28 @@ class OopsyLiveList {
   MakeRow() {
     let div = document.createElement('div');
     div.classList.add('mistake-row');
+
+    // click-to-copy function
+    div.addEventListener('click', (e) => {
+      if (this.options.ClickToCopy) {
```

Comment:
small style nit, but can you make this `if (!this.options.ClickToCopy) return;` and "early out" so that the rest of the function doesn't have to be indented?

Reply:
I just removed option.